src/components/pagination-list/pagination-list.tsx

```diff
@@ -0,0 +1,55 @@
+import styles from './pagination-list.module.css'
+
+import { PropsWithChildren, ReactNode, useRef } from 'react'
+import cn, { Argument } from 'classnames'
+
+import { t } from '@i18n'
+import { PaginationFooter } from '@components/pagination-list/components/pagination-footer'
+import { usePagination } from '@components/pagination-list/pagination-list.hook'
+
+interface IPaginationListProps<T> {
+  defaultCount?: number
```

Comment:
Do you use this 'defaultCount' somehow? And 'defaultCount' is not an obious name, maybe better 'defaultItemsCount'?

Reply:
Delete it